ci/localnet-sanity.sh

```diff
@@ -156,19 +156,7 @@ startNodes() {
   initCompleteFiles=()
   maybeExpectedGenesisHash=
   for i in $(seq 0 $((${#nodes[@]} - 1))); do
-    declare cmd=${nodes[$i]}
-
-    declare initCompleteFile="init-complete-node$i.log"
-    rm -f "$initCompleteFile"
-    initCompleteFiles+=("$initCompleteFile")
-
-    startNode "$i" "$cmd $maybeExpectedGenesisHash"
-    if $addLogs; then
-      logs+=("$(getNodeLogFile "$i" "$cmd")")
-    fi
-
-    # 1 == bootstrap validator, wait until it boots before starting
-    # other validators
+    # wait for bootstrap validator to boot before starting other validators
     if [[ "$i" -eq 1 ]]; then
```

Comment:
Maybe move this whole if block above the loop?

Reply:
would you mind explaining that in a bit more detail?

Reply:
I'm not sure I follow either - the loop initializes all validators; we just need to do different stuff for validators `> 0`